kroxylicious-kms-provider-aws-kms-test-support/src/main/java/io/kroxylicious/kms/provider/aws/kms/AbstractAwsKmsTestKmsFacade.java

```diff
@@ -105,50 +106,6 @@ public final TestKekManager getTestKekManager() {
     class AwsKmsTestKekManager implements TestKekManager {
         @Override
         public void generateKek(String alias) {
-            Objects.requireNonNull(alias);
```

Comment:
Niggle: Similar comment here.  I think the diff would have been easier to reason about if you'd maintained the order of the public methods.  I _think_ that would have resulted in a shorter diff that's really easy to reason about.